.github/workflows/ci.yml

```diff
@@ -29,32 +29,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version:
-        - "3.9"
-        - "3.10"
-        - "3.11"
-        - "3.12"
+        environment:
+        - "py39"
+        - "py310"
+        - "py311"
+        - "py312"
       fail-fast: false
     steps:
 
     - uses: actions/checkout@v3
 
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v4
+    - name: Setup pixi
+      uses: prefix-dev/setup-pixi@v0.8.14
       with:
-        python-version: ${{ matrix.python-version }}
-        cache: 'pip'
-
-    - name: Install
-      shell: bash -l {0}
-      run: source continuous_integration/scripts/install.sh
-
-    - name: Install test requirements
-      shell: bash -l {0}
-      run: |
-        set -vxeuo pipefail
-        python -m pip install .[dev]
-        python -m pip list
+        environments: ${{ matrix.environment }}
```

Comment:
I would add
```
cache: false
```
here. This way, the environment gets rebuilt for each CI run rather than re-use a different commit for `tiled`. I think this is an issue when using the following combination:
```
cache: true (the default)
```
and
```
tiled = { path = ".", editable = true }
```

Also, you can add
```
  activate_environment: true
```
so you don't have to use `pixi run -e dev` everywhere after this.